e2e-test/business/factories/MultiSigDAOFactory.ts

```diff
@@ -20,8 +27,17 @@ const UPGRADE_SAFE_LOGIC_IMPL = "upgradeSafeLogicAddress";
 const UPGRADE_DAO_LOGIC_IMPL = "upgradeDaoLogicAddress";
 const UPGRADE_SAFE_FACTORY_LOGIC_IMPL = "upgradeSafeFactoryAddress";
 
+export const DEFAULT_DAO_CONFIG = {
+  daoTreasurer: clientsInfo.treasureId.toSolidityAddress(),
```

Comment:
Please use something like that, rather passing hardcoded value here 
clientsInfo.treasureId.toSolidityAddress()
```suggestion
  daoTreasurer: "0x000000000000000000000000000000000000eef3",
```
```suggestion
  daoTreasurer: "0x000000000000000000000000000000000000eef3",
```

Reply:
yeah I changed it last commit.